From 7e2c90b4c9bcb35a81a737eb50b4720f52b79fa2 Mon Sep 17 00:00:00 2001
From: Karla Felix <karla.karolinenogueirafelix@windriver.com>
Date: Wed, 18 Oct 2023 19:42:53 -0300
Subject: [PATCH] Override docker-registry service

Prevent the docker-registry from starting when package is installed.

Signed-off-by: Charles Short <charles.short@windriver.com>
---
 debian/rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules b/debian/rules
index 02ce7be..32ef06f 100755
--- a/debian/rules
+++ b/debian/rules
@@ -10,6 +10,9 @@ export DH_GOLANG_EXCLUDES := \
 %:
 	dh $@ --buildsystem=golang --with=golang
 
+override_dh_installsystemd:
+	dh_installsystemd -p docker-registry --no-enable docker-registry.service
+
 override_dh_auto_build:
 	dh_auto_build -- -ldflags '-X github.com/docker/distribution/version.Version=$(DEB_VERSION_UPSTREAM)'
 
-- 
2.25.1

